src/UI/Components/SingleTextSelect/SingleTextSelect.tsx

```diff
@@ -145,7 +150,7 @@ export const SingleTextSelect: React.FC<Props> = ({
       default:
         if (value && value !== "no results") {
           setFilterValue("");
-          setInputValue(getDisplayValue(value) as string);
+          setInputValue(getDisplayValue(value as string));
```

Comment:
Instead of casting it as a string we could  use `String(value)`. Light suggestions as I'm fine with both ways.

Reply:
updated in other places in the file too where it was being casted